code/game/turfs/simulated/floor_icon.dm

```diff
@@ -1,4 +1,4 @@
-GLOBAL_LIST_EMPTY(flooring_cache)
+var/list/flooring_cache = list()
```

Comment:
Performance, or just a "why would anyone possibly want to try editing something in a cache"?

Reply:
Performance will be the same, though in general I don't see a reason for cache stuff to be in GLOB. I can put it back if preferred.